src/main/resources/graphql/schema.graphqls

```diff
@@ -117,4 +117,6 @@ enum RewardChangeReason {
     The associated contents are the content that are due for repetition.
     """
     CONTENT_DUE_FOR_REPETITION,
+    COMPOSITE_VALUE,
+    DONE_VALUE,
```

Comment:
I would remove this and only use composite value. What does "DONE_VALUE" even mean? I think "COMPOSITE_VALUE" is fine for both calculations.

Reply:
done